### PR TITLE
[Feature] different start mode for k8s deploy

### PR DESCRIPTION
**Pull request by shileifu**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [x] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docker/dockerfiles/fe/fe_entrypoint.sh [hunk 1]
```diff
@@ -297,7 +297,11 @@ if [[ "x$svc_name" == "x" ]] ; then
     exit 1
 fi
 
-update_conf_from_configmap
-collect_env_info
-probe_leader $svc_name
-start_fe $svc_name
+if [[ -f "/opt/starrocks/fe/meta/image/ROLE" ]];then
```
Comment (kevincai) [inline docker/dockerfiles/fe/fe_entrypoint.sh:300]: keep the same indention.

Comment (kevincai) [inline docker/dockerfiles/fe/fe_entrypoint.sh:300]: add a space after `;`.

File: docker/dockerfiles/fe/fe_entrypoint.sh [hunk 2]
```diff
@@ -297,7 +297,11 @@ if [[ "x$svc_name" == "x" ]] ; then
...
+  $STARROCKS_HOME/bin/start_fe.sh
```
Comment (kevincai) [inline docker/dockerfiles/fe/fe_entrypoint.sh:301]: still need update the conf from configmap

**Review by kevincai — COMMENTED** [review]
PR is still a draft.

**Review by kevincai — DISMISSED** [review]

**Review by kevincai — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Review by sduzh — APPROVED** [review]

**Comment by kevincai** [comment]
@mergifyio backport branch-3.0

**Pull request merged**